solutions/project/project-sandbox/setters.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setters
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  # Billing Account ID to be associated with this project
+  billing-id: "AAAAAA-BBBBBB-CCCCCC"
```

Comment:
can this value be different than the billing-id in the parent package `landing-zone-v2/setters.yaml`?

if so, this might be a case where it would get overwritten during `kpt fn render`?

from documention:
*render executes the pipelines in the package hierarchy in a depth-first order. For example, if a package A contains subpackage B, then the pipeline in B is executed on resources in B and then the pipeline in A is executed on resources in A and the output of the pipeline from package B. The output of the pipeline from A is then written to the local filesystem in-place.*
https://kpt.dev/reference/cli/fn/render/

Reply:
you are right, I may call it project-billing-id

Reply:
@alaincormier-ssc fixed now, please resolve if you agree